main.nf

```diff
@@ -205,7 +205,7 @@ bundle_for_average
     .groupTuple(by: 0)
     .set{all_bundle_for_average}
 process Average_Bundles {
-    publishDir = "./results_rbx/Average_Bundles/"
+    publishDir = params.Average_Bundles_Publish_Dir
```

Comment:
I think that we have an unwritten convention of params always being in lower case?
@GuillaumeTh

Reply:
No that's ok. We do the same thig in TractoFlow